### ISSUE TITLE
Update the version of Qiskit to 0.32

### ISSUE THREAD

**Issue by toncho11** [pyRiemann/pyRiemann-qiskit#48]
Hi,

I think we should update to the latest version of Qiskit. 
This might help with a problem where the execution of QSVM is very slow.

**Comment by gcattan** [comment]
Agree. @qbarthelemy I expect this do be a huge refactoring. Do you think we can freeze the code, and release 0.0.1 dev before doing this?

**Comment by gcattan** [comment]
Related to https://github.com/pyRiemann/pyRiemann-qiskit/issues/15

**Comment by qbarthelemy** [comment]
Yes, it's a good idea. But I don’t have much time.

Our BDFL, @sylvchev, is ok to give you maintainer rights so as not to slow down pyriemann-qiskit developments. Congrats!

**Comment by gcattan** [comment]
Link to the migration guide: 
https://github.com/Qiskit/qiskit-aqua#migration-guide

**Usages**:

In [pyriemann_qiskit/utils/hyper_params_factory.py](https://github.com/pyRiemann/pyRiemann-qiskit/blob/769605bbc71eea8ba30137ebb2aaafc60e8800b2/pyriemann_qiskit/utils/hyper_params_factory.py):

```
from qiskit.circuit.library import ZZFeatureMap, TwoLocal
from qiskit.aqua.components.optimizers import SPSA
```

In [pyriemann_qiskit/classification.py](https://github.com/pyRiemann/pyRiemann-qiskit/blob/4e739381a0a216c33d5742c420bdad05b97ae5d7/pyriemann_qiskit/classification.py)

```
from qiskit import BasicAer, IBMQ
from qiskit.aqua import QuantumInstance, aqua_globals
from qiskit.aqua.quantum_instance import logger
from qiskit.aqua.algorithms import QSVM, SklearnSVM, VQC
```
*Note*: Preferrable to address in a separate issue as QSVM was refactoring in SVM + quantum kernel.

In [pyriemann_qiskit/datasets/utils.py](https://github.com/pyRiemann/pyRiemann-qiskit/blob/a5eda74e33c4aef062e5a559597285e6007dae81/pyriemann_qiskit/datasets/utils.py)

```
from qiskit.ml.datasets import ad_hoc_data
```

In [setup.py](https://github.com/pyRiemann/pyRiemann-qiskit/blob/main/setup.py), [requirements.txt](https://github.com/pyRiemann/pyRiemann-qiskit/blob/main/requirements.txt) and [doc/requirements.txt](https://github.com/pyRiemann/pyRiemann-qiskit/blob/main/doc/requirements.txt)

```
install_requires=[ 'qiskit==0.20.0']
```
